iphone/Maps/OMaps.plist

```diff
@@ -81,6 +81,7 @@
 	<array>
 		<string>ms-outlook</string>
 		<string>googlegmail</string>
+		<string>wunderlinq</string>
```

Comment:
Why do you think OM needs this line? Is there a CanOpenURL call for the backurl in some place?

Reply:
Hi, 

This is required to launch our URL when sent via the backurl argument. CanOpenURL is used here: 

[- (void)goBack {](https://github.com/organicmaps/organicmaps/blob/850fcfdad555723dccdf92d65a69c4995264c3f6/iphone/Maps/Classes/MapViewController.mm#L723)

Reply:
That's not a flexible approach. Apps should be able to go back without us adding each one of them. What if the canOpenUrl call will be removed? Will it work?

Reply:
I'll try without canOpenUrl and will report back

Reply:
It worked.  I'm going to create a new clean branch.